### PR TITLE
[1LP][RFR] Fix test service orchestration template

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
### PRT Run

{{pytest: cfme/tests/services/test_orchestration_template.py::test_orchestration_template_crud --long-running }}

### CODE REVIEW

File: cfme/services/catalogs/orchestration_template.py [hunk 1]
```diff
@@ -173,8 +173,12 @@ def update(self, updates):
                    'content': updates.get('content')})
         view.save_button.click()
         view.flash.wait_displayed("10s")
-        view.flash.assert_success_message('Orchestration Template "{}" was saved'.format(
-            self.template_name))
+        if self.appliance.version < "5.11":
+            view.flash.assert_success_message('Orchestration Template "{}" was saved'.format(
```
[the remark this answers is not in the document]
Comment (Gauravtalreja1) [reply cfme/services/catalogs/orchestration_template.py]: DONE

File: cfme/services/catalogs/orchestration_template.py [hunk 2]
```diff
@@ -173,8 +173,12 @@ def update(self, updates):
...
+                self.template_name))
+        else:
+            view.flash.assert_success_message('Orchestration Template {} was saved'.format(
```
[the remark this answers is not in the document]
Comment (Gauravtalreja1) [reply cfme/services/catalogs/orchestration_template.py]: DONE

**Comment by dajoRH** [comment]
Lint report for commit 14af835e2e102ad18fb6365ce7313e4fa58fbe97:

`cfme/services/catalogs/orchestration_template.py`:
- :bangbang: Line 177:101: **E501** *line too long (105 > 100 characters)*
- :bangbang: Line 179:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit b86e23e444991922ce2bfe78ec2eab28cca6bfdb:

`cfme/services/catalogs/orchestration_template.py`:
- :bangbang: Line 177:101: **E501** *line too long (105 > 100 characters)*
- :bangbang: Line 179:101: **E501** *line too long (103 > 100 characters)*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit d4cbde15d427619a5552eddb308ea5bc85ace74c:

`cfme/services/catalogs/orchestration_template.py`:
- :bangbang: Line 177:11: **E111** *indentation is not a multiple of four*
- :bangbang: Line 180:11: **E111** *indentation is not a multiple of four*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dgaikwad** [comment]
LGTM. Thank you for the fix.

**Pull request merged**